cli/tests/it/cmd.rs

```diff
@@ -734,3 +740,162 @@ contract CTest is DSTest {
     let cache_after = fs::read_to_string(prj.cache_path()).unwrap();
     assert_eq!(cache, cache_after);
 });
+
+forgetest_async!(
+    can_deploy_script_without_lib,
+    (|prj: TestProject, cmd: TestCommand| async move {
+        let port = next_port();
+        spawn(NodeConfig::test().with_port(port)).await;
+        let mut tester = ScriptTester::new(cmd, port, prj.root());
+
+        tester
+            .load_private_keys(vec![0, 1])
+            .await
+            .add_sig("BroadcastTestNoLinking", "deployDoesntPanic()")
+            .simulate("SIMULATION COMPLETE. To broadcast these")
```

Comment:
consider letting `simulate` take some kind of closure that represents the `Outcome` and this type then knows how to check for `"SIMULATION COMPLETE. To broadcast these"` so we don't have this all over the place and can make changes in one place